.changeset/clean-timers-argue.md

```diff
@@ -0,0 +1,6 @@
+---
+"@blobscan/api": patch
+"@blobscan/web": patch
```

Comment:
```suggestion
"@blobscan/api": minor
"@blobscan/web": minor
```

Reply:
I think this qualifies more as a minor update than a patch, since we're adding a new field to the blob model, thereby introducing a new feature that maintains backward compatibility.